src/sparseml/export/validators.py

```diff
@@ -86,6 +92,18 @@ def validate_structure(
             raise FileNotFoundError(f"File {file_path} is missing.")
 
 
+def validate_structure_external_data(
+    deployment_directory_path: Union[str, Path], onnx_model_name: Union[str, Path]
+):
+    files_present = onnx_data_files(
+        onnx_model_name.replace(".onnx", ".data"), deployment_directory_path
```

Comment:
nit: since we end up doing this replace twice could we just add it in as part of the onnx_data_files function?

Reply:
I would like to keep those two concerns: creation of the `onnx_data_name` variable and looking up data name files, separate